client/my-sites/stats/site.jsx

```diff
@@ -267,23 +265,75 @@ class StatsSite extends Component {
 					</div>
 				</div>
 				<JetpackColophon />
+			</>
+		);
+	}
+
+	renderEnableStatsModule() {
+		const { slug, siteId, path } = this.props;
+		return (
+			<EmptyContent
+				illustration="/calypso/images/illustrations/illustration-404.svg"
+				title={ translate( 'Looking for Stats?' ) }
+				line={ translate(
+					'Enable Site Stats below or visit {{a}}Traffic > Site Stats{{/a}} for more options',
+					{
+						components: {
+							a: <a href={ `/marketing/traffic/${ slug }` } />,
+						},
+					}
+				) }
+				action={ translate( 'Enable Site Stats' ) }
+				actionCallback={ () => this.props.enableJetpackStatsModule( siteId, path ) }
+			/>
+		);
+	}
+
+	render() {
+		const { isJetpack, siteId, showEnableStatsModule } = this.props;
+
+		const { period } = this.props.period;
+
+		return (
+			<Main wideLayout={ true }>
+				<QueryKeyringConnections />
+				{ isJetpack && siteId && <QueryJetpackModules siteId={ siteId } /> }
+				{ siteId && <QuerySiteKeyrings siteId={ siteId } /> }
+				<DocumentHead title={ translate( 'Stats and Insights' ) } />
+				<PageViewTracker
+					path={ `/stats/${ period }/:site` }
+					title={ `Stats > ${ titlecase( period ) }` }
+				/>
+				{ showEnableStatsModule ? this.renderEnableStatsModule() : this.renderStats() }
 			</Main>
 		);
 	}
 }
+const enableJetpackStatsModule = ( siteId, path ) =>
+	withAnalytics(
+		recordTracksEvent( 'calypso_jetpack_module_toggle', {
```

Comment:
I think this event doesn't make that much sense, given that this isn't really triggered by a toggle. Should we leave the `calypso_jetpack_module_toggle` event name to the `JetpackModuleToggle` component and use another one here? 

Or ideally, should we have a generic module (de)activation event that is handled in the data layer? Likely not as part of this PR, though.

Reply:
I think this is the only way we track module activation/deactivation events currently, so I mostly mirrored the tracks event here in case the information would be interest. The path property is enough to distinguish where this is coming from, but I'm also happy to remove this fully if folks don't think it's interesting.

> Or ideally, should we have a generic module (de)activation event that is handled in the data layer?

This would probably make more sense provided we didn't mind the gap in history :D

Reply:
I'll keep this stats call for now unless folks had strong feelings here. Should be easy enough to audit in a refactor.